packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -748,10 +767,17 @@ export class VerticalStackedBarChartBase extends React.Component<
       const xPoint = xBarScale(
         this._xAxisType === XAxisTypes.NumericAxis
           ? (singleChartData.xAxisPoint as number)
+          : this._xAxisType === XAxisTypes.DateAxis
+          ? (singleChartData.xAxisPoint as Date)
           : (singleChartData.xAxisPoint as string),
       );
       const xScaleBandwidthTranslate =
-        this._xAxisType === XAxisTypes.NumericAxis ? 0 : (xBarScale.bandwidth() - this._barWidth) / 2;
+        this._xAxisType === XAxisTypes.NumericAxis
+          ? 0
```

Comment:
>0 [](http://example.com/codeflow?start=12&length=1)

shouldn't this behave same as numeric scale

Reply:
I tried same using similar logic as numeric scale but bars are not coming properly. Time Range is also +/-barwidth/2 , so we have to adjust that while making bars.

Reply:
could you mention this as comment in the code

Reply:
Done